requirements.txt

```diff
@@ -4,18 +4,17 @@ joblib==0.11
 matplotlib==2.1.2
 nose==1.3.7
 notebook==5.7.2
-numpy==1.14.*
+numpy==1.14.0
```

Comment:
Why this change?

Reply:
I don't have strong feelings about this, but I'm always paranoid about differences in even minor version of `numpy` leading to small differences in results.